### DISCUSSION TITLE
Is it possible to make playback stop when it reaches the end of the region?

### DISCUSSION THREAD

**Discussion by johtso** [mifi/lossless-cut#1427] [Q&A]
When cropping some audio, I always want to preview what it's going to sound like before I've exported it (did I chop too much off the end?)

I can't find any way to get playback to stop once it reaches the end of the region. Am I missing something?

**Comment by mifi** [comment]
I believe what you're asking is the same as one of the points here: #253

**Reply by johtso** [reply]
#574 as well

I understand it's a little tricky with video because of keyframes, but with simple audio that's much less of an issue and you just want to be able to hear the segment you've selected.

I'm currently at the point where I'm about to write a tiny tool with javascript, wavesurferjs and the regions plugin.. just to select and preview sections of a audio file and feed those cut points into ffmpeg. It would be really cool if lossless-cut lent itself better to low friction cropping workflows with audio. I really appreciate it for all the things it does so well though!

**Reply by mifi** [reply]
yes for audio it definitely makes sense to have such a feature! I'll see how hard it is to implement

**Reply by mifi** [reply]
this is now implemented
